.github/workflows/main.yml

```diff
@@ -18,3 +18,12 @@ jobs:
       - name: Run benchmark
         run: |
           bash ./.github/scripts/run-nightly-nodocker.sh
+      - name: Compress artifact
+        run: |
+          export TODAY=$(date "+%Y%m%d")
```

Comment:
is it better to include hour/min/sec to this, in case some day this job runs twice for some reason?

Reply:
Good catch!

Reply:
Good point!